src/start/RealmGate.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import connectWithActions from '../connectWithActions';
+import type { Actions } from '../types';
+import { Screen, Centerer, ZulipButton } from '../common';
+
+const componentStyles = StyleSheet.create({
+  divider: {
+    height: 20,
+  },
+});
+
+type Props = {
+  actions: Actions,
+};
+
+class RealmGate extends PureComponent<Props> {
```

Comment:
Let's call this `WelcomeScreen`. It isn't obvious to me looking at this what "gate" means here; and I think ending the names of whole-screen components like this with "Screen" is helpful.

Reply:
Okay, will change that.